backend/databasemodel/model.sql

```diff
@@ -1449,12 +1462,13 @@ ALTER TABLE kooste.tamperewfs_luonnonmuistomerkit OWNER TO tarmo_admin;
 CREATE TABLE kooste.tamperewfs_luontopolkurastit (
 	mi_prinx bigint NOT NULL,
 	geom geometry(MULTIPOINT, 4326) NOT NULL,
-	nakyvyys boolean NOT NULL DEFAULT True,
-	nimi text,
+	visibility boolean DEFAULT True,
+	name text NOT NULL,
+	tarmo_category text DEFAULT 'Ulkoilureitit',
+	type_name text DEFAULT 'Luontopolkurasti',
 	tunnus integer,
 	rasti integer,
-	kohteenkuvaus text,
-	lisatietoja text,
```

Comment:
Are we sure we don't want the `lisatietoja` field at all? Perhaps better to retain the extra data just in case? I guess `kohteenkuvaus` is the one that should be copied to `infoFi`?

Reply:
Looks like that the idea was not to remove fields, because upgrade.sql does not remove fields?

Reply:
Yes, kohteenkuvaus will be renamed to infoFi (see row 213 in upgrade.sql). I changed my mind about the relevance of that lisatietoja field since in principle it should be renamed to additionalInfo but that field was meant for storing notes of tarmo admin so we really have no place / reason to store that information. What did you mean by upgrade not removing the fields? I guess lisatietoja field is removed in row 216 in upgrade.sql?

Reply:
I guess the main reason was to try to make data structures of all tables as similar as possible (in oreder to make UI desing easier). But it seems like lisatietoja field contains some valuable information so let's keep it.